quality_control_stock/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     "name": "Quality control - Stock",
-    "version": "8.0.1.0.1",
+    "version": "8.0.1.0.2",
```

Comment:
you are not making a migration, why changing the version?

Reply:
Changed the theme of the version. Thanks.